lib/membrane_opus/description.ex

```diff
@@ -1,42 +1,37 @@
-defmodule Membrane.Caps.Audio.Opus do
+defmodule Membrane.Opus.Description do
   @moduledoc """
-  This module implements struct for caps representing a single Opus-encoded
-  audio frame.
+  This module implements struct describing a single Opus-encoded audio frame.
   """
 
   # Opus frame duration type.
   #
   # 2 means 2.5ms, other values just map into duration in miliseconds.
   @type frame_duration_t :: 2 | 5 | 10 | 20 | 40 | 60
 
-
   # Amount of channels inside Opus frame.
   #
   # Can be 1 or 2.
   @type channels_t :: 1 | 2
 
-
   # Enable/disable inband forward error correction
   #
   # Can be true (FEC enabled) or false (FEC disabled)
   @type enable_fec_t :: boolean
```

Comment:
Since it's stream description maybe `fec_enabled(_t)` would be more appropriate? Or `forward_error_correction?`